doc/sailfish-webview.cpp

```diff
@@ -0,0 +1,31 @@
+namespace SailfishOS {
+namespace WebView {
+}
+}
+
+/*!
+  \module SailfishOAuth
```

Comment:
typo

Reply:
Thanks; fixed, plus a couple of others as well.